app/scripts/controllers/transactions/tx-state-manager.js

```diff
@@ -81,7 +86,7 @@ export default class TransactionStateManager extends EventEmitter {
    * @returns {TransactionMeta} the default txMeta object
    */
   generateTxMeta(opts = {}) {
-    const netId = this.getNetwork();
+    const netId = this.getNetworkState();
```

Comment:
Side note, we should make these variables consistent at some point too. It's confusing that sometimes this is `netId`, sometimes, `network`, sometimes `networkState`.

Reply:
The value itself is confusing, even apart from the name. It's a mix of "loading" and network ID. We use it in some places _calling it_ "network ID", but using it just for the "loading" part.

Clarifying this is on the API normalization list.